### PR TITLE
Add postback trigger to the webhooks documentation

### PR DESCRIPTION
**Pull request by Mario54**
@alavers @mspensieri @jugarrit @lemieux

### CODE REVIEW

File: source/rest/index.md [hunk 1]
```diff
@@ -169,51 +169,11 @@ The `jwt` payload must include a `scope` claim which specifies the caller's scop
...
-              "osVersion": "9.2"
-            }
-          }
-        ]
-    }
-}
-```
-
-Webhooks are a fantastic way to extend the Smooch platform beyond the built-in feature set. You can use webhooks to build your own Smooch chat clients or to integrate more deeply with your favorite CRM.
+Webhooks are a fantastic way to extend the Smooch platform beyond the built-in feature set. You can use webhooks to build your own Smooch chat clients, to integrate more deeply with your favorite CRM, or to build a bot.
```
Comment (alavers) [inline source/rest/index.md]: why not linkify "build a bot" into a shameless plug for smooch/smooch-bot? :D

**Comment by alavers** [comment]
👍

File: source/rest/index.md (hunk 1 above)
Comment (Mario54) [inline source/rest/index.md]: good idea 😄

File: source/rest/index.md [hunk 2]
```diff
@@ -437,11 +397,106 @@ smooch.webhooks.create({
...
+{
+    "trigger": "message:appUser",
+    "messages":[{
+        "_id": "55c8c1498590aa1900b9b9b1",
+        "text": "Hi! Do you have time to chat?",
+        "role": "appUser",
+        "authorId": "c7f6e6d6c3a637261bd9656f",
+        "name": "Steve",
+        "received": 1444348338.704,
+        "metadata": [],
```
Comment (mspensieri) [inline source/rest/index.md]: Metadata is supposed to be an object

**Comment by mspensieri** [comment]
👍

**Comment by dannytranlx** [comment]
👍

File: source/rest/index.md [hunk 3]
```diff
@@ -437,11 +397,106 @@ smooch.webhooks.create({
 
 A webhook will make a request to the target each time a trigger associated with the webhook occurs. Triggers are specified in an optional `triggers` array in the request body. If `triggers` is not specified the webhook will be configured with the `message` trigger by default.
 
+A webhook with a `postback` trigger will be fired every time a user clicks on [an action button with type `postback`](#action-buttons).
```
Comment (dannytranlx) [inline source/rest/index.md]: Where is `#action-buttons`? On the regular docs?

Comment (Mario54) [inline source/rest/index.md]: It was added in #94

Comment (dannytranlx) [inline source/rest/index.md]: Oh, my b!

**Comment by dannytranlx** [comment]
👍

**Pull request merged**